lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs

```diff
@@ -30,18 +30,18 @@ pub const CHANNEL_NAME: &str = "cliprdr";
 /// https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpeclip/fb9b7e0b-6db4-41c2-b83c-f889c1ee7688
 pub struct Client {
     clipboard: HashMap<u32, Vec<u8>>,
-    on_remote_copy: Box<dyn Fn(Vec<u8>)>,
+    on_remote_copy: Box<dyn Fn(Vec<u8>) -> RdpResult<()>>,
```

Comment:
Why does `on_remote_copy` now need to return a result? What changed about the function that it can now fail?